app/assets/javascripts/application/tools/congress_message.js

```diff
@@ -1,6 +1,23 @@
 $(document).on("ready", function() {
   if ($("#congress-message-tool").length){  // showing paging at top of form
     $(".page-indicator").css("display", "grid");
+    if ($(".page-indicator").children().length == 3) {
+      $(".page-indicator").css({
+        "grid-template-columns": "33% 33% 33%"
+      });
+    }
+  }
+
+  if ($(".load-target-members").length && $(".load-target-members").is(":visible")) {
```

Comment:
The `.length` check here looks redundant -- if the element is missing, it shouldn't be visible.